charts/graphistry-helm/values.yaml

```diff
@@ -76,6 +79,16 @@ multiNode: false
 # rolling update deployment strategy switch (postgres and nexus are exempt)
 rollingUpdate: false
 
+# django session cookie timeout (seconds)
+sessionCookieAge : "1209600"
+
+# django drf-jwt, jwt token timeout (seconds)
+jwtExpirationDelta : "3600"
+
+# ENABLE django silk - performance analysis library
+enableDjangoSilk : "False"
+
+
 #restart policy
 restartPolicy: Always
```

Comment:
I think also an equiv for `COOKIE_SECURE=True` (default off,TLS should generally flip on auto or at least manually)

Reply:
added here: https://github.com/graphistry/graphistry-helm/pull/20/commits/3877b1522a65d3dedc7c44d5b6e1b5fa01b388fb